src/js/SubmissionContainer.jsx

```diff
@@ -52,22 +53,26 @@ var SubmissionContainer = React.createClass({
       )
     }
 
-    if(code > 3) progress = <p>Progress component goes here</p>
+    if(code > 3) progress = <p className="third">Progress component goes here</p>
 
-    if(code > 6) editsContainer = <EditsContainer/>
+    if(code > 5) editsContainer = <EditsContainer/>
```

Comment:
I changed this because `6: Validation begins`, so wouldn't we want to start to be able to show edits right away?